pom.xml

```diff
@@ -114,192 +195,491 @@
             <plugins>
                 <plugin>
```

Comment:
You missed semantic versioning plugin as well as the dependencies.

Reply:
I think it is good to put these in the parent pom.

Reply:
Most likely. I didn't include it now on purpose because the plugin required a specific file in each project and I didn't extract that to a common piece yet.

I think we can do that next.